Reference cards

jmthompson2015 opened the pull request:
Added missing reference card images. Used TinyPNG on images.

guidokessels commented on data/reference-cards.js:
```diff
@@ -1,160 +1,176 @@
+      "id": 14,
+      "title": "Tractor Beam Token",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/TractorBeanToken.png"
```
This needs to be changed to `reference-cards/TractorBeamToken.png` as you renamed the image.

guidokessels replied:
This is why the build is failing.

guidokessels commented:
Thanks for this! There's a small error that needs to be fixed to make the build pass. I added a comment to the line that needs to be changed.

Could you also check the whitespace? Right now the git diff shows that every line has changed because the file's indenting has changed.

jmthompson2015 commented:
I fixed the filename and the whitespace issue (turned atom-beautify off). And added a couple more reference cards while I was at it. Please take another look.

guidokessels commented:
Thanks a lot!
